src/s1reader/s1_annotation.py

```diff
@@ -7,20 +7,75 @@
 
 import datetime
 import os
-import lxml.etree as ET
 import zipfile
 
+from types import SimpleNamespace
+
+import lxml.etree as ET
 import numpy as np
 
-from scipy.interpolate import InterpolatedUnivariateSpline, interp1d
+from isce3.core import speed_of_light
 from packaging import version
+from scipy.interpolate import InterpolatedUnivariateSpline, interp1d
 
-from isce3.core import speed_of_light
 
 # Minimum IPF version from which the S1 product's Noise Annotation
 # Data Set (NADS) includes azimuth noise vector annotation
 min_ipf_version_az_noise_vector = version.parse('2.90')
 
+# Minimum IPF version from which the RFI information gets available
```

Comment:
Can we add a link to a document where this is stated?

Reply:
Good idea. I've added few information about the document I've referenced.